### DISCUSSION TITLE
Authentication MTLS?

### DISCUSSION THREAD

**Discussion by elohim85** [mockoon/mockoon#399] [Help] [answered]
With this API, when I enable HTTPS, it is possible for it to work as MTLS?

**Comment by 255kb** [comment, marked as answer]
Currently, Mockoon is using a basic self-signed certificate. No other options are available. 
However, providing your own certificates is on the roadmap https://github.com/mockoon/mockoon/issues/175 (no ETA yet).
I will probably expose Node's TLS options as is (`ca`, `cert`, `key`, `rejectUnauthorized`, `requestCert) so you can setup mTLS easily.

**Reply by elohim85** [reply]
about of the self-signed certificate, can I get its public key?

Thanks

**Reply by 255kb** [reply]
you will find everything there: https://github.com/mockoon/commons/blob/main/src/constants/ssl.constants.ts

**Comment by elohim85** [comment]
Okay, that'd be great.

Very thanks
